efel/units/units.json

```diff
@@ -0,0 +1,129 @@
+{
+    "ADP_peak_amplitude": "mV",
+    "ADP_peak_values": "mV",
+    "AHP_depth": "mV",
+    "AHP_depth_abs": "mV",
+    "AHP_depth_abs_slow": "mV",
+    "AHP_depth_diff": "mV",
+    "AHP_depth_from_peak": "mV",
+    "AHP_depth_last": "mV",
+    "AHP_depth_slow": "mV",
+    "AHP_slow_time": "constant",
+    "AHP_time_from_peak": "ms",
+    "AHP_time_from_peak_last": "ms",
+    "AP1_peak": "mV",
+    "AP2_AP1_begin_width_diff": "ms",
+    "AP2_AP1_diff": "mV",
+    "AP2_AP1_peak_diff": "mV",
+    "AP_Amplitude": "mV",
+    "AP_Amplitude_change": "constant",
+    "AP_amplitude_diff": "mV",
+    "AP_amplitude_from_voltagebase": "mV",
+    "AP_begin_time": "ms",
+    "AP_begin_voltage": "mV",
+    "AP_begin_width": "ms",
+    "AP_duration": "ms",
+    "AP_duration_change": "constant",
+    "AP_duration_half_width": "ms",
+    "AP_duration_half_width_change": "constant",
+    "AP_fall_rate": "V/s",
+    "AP_fall_rate_change": "constant",
+    "AP_fall_time": "ms",
+    "AP_peak_downstroke": "V/s",
+    "AP_peak_upstroke": "V/s",
+    "AP_phaseslope": "1/(ms)",
+    "AP_phaseslope_AIS": "1/(ms)",
+    "AP_rise_rate": "V/s",
+    "AP_rise_rate_change": "constant",
+    "AP_rise_time": "ms",
+    "AP_width": "ms",
+    "AP_width_between_threshold": "ms",
+    "BAC_maximum_voltage": "mV",
+    "BAC_width": "ms",
+    "BPAPAmplitudeLoc1": "mV",
+    "BPAPAmplitudeLoc2": "mV",
+    "BPAPHeightLoc1": "mV",
+    "BPAPHeightLoc2": "mV",
+    "ISI_CV": "constant",
+    "ISI_log_slope": "ms",
+    "ISI_log_slope_skip": "ms",
+    "ISI_semilog_slope": "ms",
+    "ISI_values": "ms",
+    "Spikecount": "constant",
+    "Spikecount_stimint": "constant",
+    "adaptation_index": "constant",
+    "adaptation_index_2": "constant",
+    "all_ISI_values": "ms",
+    "amp_drop_first_last": "mV",
+    "amp_drop_first_second": "mV",
+    "amp_drop_second_last": "mV",
+    "burst_begin_indices": "constant",
```

Comment:
where did you get value for this? I didn't put the *_indices features in the documentation because it would be confusing for the user due to the interpolation.

Nothing wrong with having them in the units file though.

Reply:
I got it from EModelRunner's sscx units.json file. Should I delete it?

Reply:
Yes, I guess you can, to be consistent with the docs

Reply:
Alright it is removed with the latest push.